package.json

```diff
@@ -26,22 +26,24 @@
     "3d"
   ],
   "workspaces": [
-    "modules/*"
+    "modules/core",
+    "modules/loaders.gl"
   ],
   "browser": {
     "fs": false
   },
   "scripts": {
+    "bootstrap": "scripts/bootstrap.sh",
     "start": "echo 'Please see luma.gl website for how to run examples' && open http://uber.github.io/luma.gl/#/documentation/getting-started/examples",
     "clean": "rm -fr dist dist-es6 && (cd modules/core && npm run clean)",
-    "build": "(cd modules/core && npm run build)",
+    "build": "lerna run build",
     "cover": "NODE_ENV=cover nyc node test/start",
     "lint": "eslint modules test && npm run lint-yarn",
     "lint-examples": "eslint examples",
     "lint-yarn": "!(grep -q unpm.u yarn.lock) || (echo 'Please rebuild yarn.lock file using public npmrc' && false)",
     "lint-docs": "scripts/lint-markdown.sh",
-    "publish-prod": "(cd modules/core && npm run publish-prod)",
-    "publish-beta": "(cd modules/core && npm run publish-beta)",
+    "publish-prod": "lerna publish --cd-version minor",
```

Comment:
loaders.gl is currently being broken out to a separate repo. Not sure if it is worth introducing combined publishing here at this stage. But perhaps no harm.

Reply:
I'll do another pass of cleaning up once they are moved.